lib/robots.txt

```diff
@@ -0,0 +1 @@
+Sitemap: {{ "sitemap.xml" | relative_url }}
```

Comment:
This should be an absolute URL.

According to [sitemaps.org](https://www.sitemaps.org/protocol.html#informing)

> #### Specifying the Sitemap location in your robots.txt file
> You can specify the location of the Sitemap using a robots.txt file. To do this, simply add the following line including the full URL to the sitemap:
> ```text
> Sitemap: http://www.example.com/sitemap.xml
> ```
> This directive is independent of the user-agent line, so it doesn't matter where you place it in your file.